src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsHopObject.java

```diff
@@ -20,9 +21,9 @@
  */
 public class AdsHopObject extends AdsBaseObject {
 
-  private static final String POST_MESSAGE_SCRIPT_XPATH =
+  private final static String POST_MESSAGE_SCRIPT_XPATH =
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)